samples/expected_output/unitive-filename-pattern/actionwords.js

```diff
@@ -0,0 +1,213 @@
+
+  },
+  sovrenCannotParseTheResume: function () {
+
+  },
+  pNGCreationFails: function () {
+
+  },
+  defaultIconOfPDFDisplays: function () {
+
+  },
+  userClicksSaveOnParsedResume: function () {
+
+  },
+  thatEmailAlreadyHasAResumeInTheSystemOnThatJob: function () {
+
+  },
+  userIsNotifiedThatThisPersonAlreadyHasAResumeForThisJobDoYouWantToUseThatOneOrTheNewOne: function () {
+
+  },
+  iClickYesReplace: function () {
+
+  },
+  theBackEndKeepsTheOriginal: function () {
+
+  },
+  iAmAJobOwnereditorAndIAmGoingToUseUnitiveResumeReviewer: function () {
+
+  },
+  aThumbnailPreviewShowsOfTheResumeInTheField: function () {
+
+  },
+  iAmUser: function () {
+
+  },
+  iHaveSavedAtLeastOneInterviewQuestionToOneInterviewerOnBuildInterviewPageOrAtLeastOneResumeReviewed: function () {
+
+  },
+  jobHas0ApplicantsApprovedForInterview: function () {
+
+  },
+  p1ButtonIsBlueAndNotWiggle: function (p1) {
+
+  },
+  imOnTheP1Page: function (p1) {
+
+  },
+  jobIsCreated: function () {
+
+  },
+  0ResumesAdded: function () {
+
+  },
+  0ApplicantsApprovedForInterview: function () {
+
+  },
+  thereAreNoQuestioninterviewersSaved: function () {
+
+  },
+  p1ShouldBePrimaryButtonAndWiggle: function (p1) {
+
+  },
+  iHaveAddedAnyApplicants: function () {
+
+  },
+  atLeastOneInterviewQuestionIsSavedRegardlessOfWhetherIveAddedOrReviewedResumes: function () {
+
+  },
+  iClickBackToJobDashboard: function () {
+
+  },
+  thereIsNotAlreadyAScheduledInterview: function () {
+
+  },
+  p1ButtonIsBlueAndNOTWiggle: function (p1) {
+
+  },
+  stepSix: function () {
+
+  },
+  aMiracleOccurs: function () {
+
+  },
+  raceYouToSeven: function () {
+
+  },
+  raceYouToEight: function () {
+
+  },
+  raceYouToNine: function () {
+
+  },
+  canYouType: function () {
+
+  },
+  raceYouToEleven: function () {
+
+  },
+  jobHas2OrMoreResumesAdded: function () {
+
+  },
+  0Reviewed: function () {
+
+  },
+  weAreOnThe: function () {
+
+  },
+  itShouldShowTheSearchBar: function () {
+
+  },
+  imOnTheJobCopyPage: function () {
+
+  },
+  itShouldShowTheListOfAvailableReqsAndTheExtraDataWithIt: function () {
+
+  },
+  itShouldShowANodataMessageWhenThereAreNoReqsAvailableToTheUser: function () {
+
+  },
+  itShouldShowANodataMessageWhenThereAreNoReqsAfterSearchingfiltering: function () {
+
+  },
+  itShouldShowTheJobDescriptionWhenClickingTheP1Button: function (p1) {
+
+  },
+  itShouldCreateACopyOfTheReqAndJobDescriptionWhenClickingTheP1Button: function (p1) {
+
+  },
+  itShouldShowTenReqsAndAShowMoreLink: function () {
+    this.itShouldShowP1Reqs("10");
+    this.itShouldShowP1Link("Show more");
+  },
+  theSetupForRequisitionCopyCtrl: function () {
+
+  },
+  itShouldShowAJobDescriptionPopup: function () {
+
+  },
+  itShouldRequestToCopyTheGivenReqAndNavigateToTheRequisitionEditPage: function () {
+
+  },
+  itShouldShowAnErrorIfItFails: function () {
+
+  },
+  itShouldShowP1Reqs: function (p1) {
+
+  },
+  itShouldShowP1Link: function (p1) {
+
+  }
+};
```

Comment:
This is the golden file representing generated code, and it is being generated without the trailing newline.